mono/metadata/unity-utils.c

```diff
@@ -1102,6 +1102,11 @@ void* mono_unity_get_field_address(MonoObject *obj, MonoVTable *vt, MonoClassFie
 	return src;
 }
 
+MONO_API MonoClassField* mono_unity_field_from_token(MonoImage *image, guint32 token, MonoClass **retklass, MonoGenericContext *context, MonoError *error)
```

Comment:
I would suggest adding the `_checked` suffix if we are exposing the error.

Reply:
Force pushed this suggestion